packages/odyssey-ie-11-sandbox/src/App.tsx

```diff
@@ -0,0 +1,518 @@
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col" format={"num"}>
+                Radius (km)
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col">Type</Table.HeaderCell>
+              <Table.HeaderCell scope="col" format={"date"}>
+                Perihelion date
+              </Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            <Table.Row>
+              <Table.DataCell>Jupiter</Table.DataCell>
+              <Table.DataCell format={"num"}>69,911</Table.DataCell>
+              <Table.DataCell>Gas giant</Table.DataCell>
+              <Table.DataCell format={"date"}>January 21, 2023</Table.DataCell>
+            </Table.Row>
+            <Table.Row>
+              <Table.DataCell>Earth</Table.DataCell>
+              <Table.DataCell format={"num"}>6,371</Table.DataCell>
+              <Table.DataCell>Terrestrial</Table.DataCell>
+              <Table.DataCell format={"date"}>January 2, 2021</Table.DataCell>
+            </Table.Row>
+            <Table.Row>
+              <Table.DataCell>Mercury</Table.DataCell>
+              <Table.DataCell format={"num"}>1,737</Table.DataCell>
+              <Table.DataCell>Terrestrial</Table.DataCell>
+              <Table.DataCell format={"date"}>&ndash;</Table.DataCell>
+            </Table.Row>
+          </Table.Body>
+        </Table>
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Tag List"
+        label="Status Label"
+        labelHidden
+        variant="neutral"
+      />
+      <ErrorBoundary>
+        <TagList tags={["Item one", "Item two", "Item three"]} />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Text"
+        label="Status Label"
+        labelHidden
+        variant="neutral"
+      />
+      <ErrorBoundary>
+        <Text>Text</Text>
+      </ErrorBoundary>
+
+      <Status
+        descriptor="TextArea"
+        label="Status Label"
+        labelHidden
+        variant="caution"
+      />
+      <ErrorBoundary>
+        <TextArea hint="Descriptive field hint" label="TextArea" />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="TextInput"
+        label="Status Label"
+        labelHidden
+        variant="caution"
+      />
+      <ErrorBoundary>
+        <TextInput label="TextInput" />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Toast"
+        label="Status Label"
+        labelHidden
+        variant="danger"
+      />
+      <ErrorBoundary>
+        <Toast
+          heading="Info Modal"
+          body="Modal text"
+          dismissButtonLabel="Close"
+        />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Tooltip"
+        label="Status Label"
+        labelHidden
+        variant="neutral"
+      />
+      <ErrorBoundary>
+        <Tooltip label="Top tooltip label" position="top">
+          <Button>Top</Button>
+        </Tooltip>
+      </ErrorBoundary>
+    </main>
+  );
+}
```

Comment:
can we restructure this in a way that isn't a 500+ line component? Harder to review and maintain into the future!

I'm also curious about rendering each component within a separate browser frame so we can rule out any type of pollution or shared DOM issues for testing. What do you think?